packages/app-builder/src/components/Scenario/Sanction/FieldSkipIfUnder.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Input, Switch } from 'ui-design-system';
+
+interface FieldSkipIfUnderProps {
+  value: number | null;
+  onBlur: () => void;
+  onChange: (value: number | null) => void;
+  editor: 'view' | 'edit';
+  name: string;
+}
+
+export const FieldSkipIfUnder = ({
+  value,
+  onBlur,
+  onChange,
+  editor,
+  name,
+}: FieldSkipIfUnderProps) => {
+  const { t } = useTranslation();
```

Comment:
issue: we miss the desired namespace
```suggestion
  const { t } = useTranslation(['scenarios']);
```